src/main/java/org/symphonyoss/integration/webhook/jira/parser/IssueUpdatedJiraParser.java

```diff
@@ -111,10 +111,12 @@ private Entity getChangeLogEntity(JsonNode node) {
     for (int i = 0; i < items.size(); i++) {
       JsonNode item = items.get(i);
 
+      String toStringAttribute = item.get(TOSTRING_PATH).isNull() ? "" : item.get(TOSTRING_PATH).asText();
```

Comment:
I didn't know we had apache-commons-lang in this project. I'm going to fix that.